### PR TITLE
Feature/scan info

### PR DESCRIPTION
**Pull request by Christiantyemele**

### CODE REVIEW

**Review by Ngha-Boris — CHANGES_REQUESTED** [review]
Hello @Christiantyemele one of the checks failed please consider checking that out.

**Comment by Christiantyemele** [comment]
> Hello @Christiantyemele one of the checks failed please consider checking that out.

@Ngha-Boris issue resolved plz review

File: package-lock.json
Comment (Ngha-Boris) [inline package-lock.json:1]: Please @Christiantyemele  remove this file. Here we're using only yarn.

Comment (Christiantyemele) [reply package-lock.json:1]: Done


> Please @Christiantyemele remove this file. Here we're using only yarn.

Done

File: src/index.scss [hunk 1]
```diff
@@ -1,3 +1,77 @@
...
+}
+
+.add:hover::after {
+  visibility: visible;
+  opacity: 1;
+}
+
+.add {
+  cursor: pointer;
+}
```
Comment (stephane-segning) [inline src/index.scss:77]: Remove this and use tailwind only

**Review by stephane-segning — CHANGES_REQUESTED** [review]

File: src/components/Scan-Button.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { Link } from 'react-router-dom';
+
+export const InfoButton = () => {
```
Comment (stephane-segning) [inline src/components/Scan-Button.tsx:3]: This component's name doesn't match the file name

File: src/app.tsx
Comment (stephane-segning) [inline src/app.tsx:1]: Cancel these changes

File: package-lock.json
Comment (stephane-segning) [inline package-lock.json:1]: Remove this file. We're using yarn here

**Review by stephane-segning — CHANGES_REQUESTED** [review]

File: package-lock.json
[the remark this answers is not in the document]
Comment (Christiantyemele) [reply package-lock.json:1]: no it has been removed plz review

Comment (Christiantyemele) [reply package-lock.json:1]: changes done please review again sir @stephane-segning

**Review by bryandino673 — APPROVED** [review]

File: src/index.scss (hunk 1 above)
Comment (Christiantyemele) [reply src/index.scss]: Done sir @stephane-segning

File: src/components/Scan-Button.tsx (hunk 2 above)
Comment (Christiantyemele) [reply src/components/Scan-Button.tsx]: Done sir @stephane-segning

**Pull request closed without merging**